arviz/stats/diagnostics.py

```diff
@@ -530,6 +535,13 @@ def _bfmi(energy):
     return num / den
 
 
+def _backtransform_ranks(arr, c=3 / 8):
+    """Backtransformation of ranks."""
```

Comment:
What does backtransformation do?

Reply:
magic

Reply:
I will update the docstring based on the posterior docstring.